vshard/router/init.lua

```diff
@@ -1546,6 +1546,8 @@ local function router_info(router, opts)
                 router.discovery_service:info(),
         }
     end
+
```

Comment:
Nit: new line is not necessary here, we don't really use them, unless it's essential to split the part of code. Same in another place